### PR TITLE
AO-373 Update example .conf files of argo-ncg

### PR DESCRIPTION
**Pull request by eimamagi**

### CODE REVIEW

File: config/ncg.conf [hunk 1]
```diff
@@ -8,30 +8,20 @@
...
-# NRPE_UI=nrpe.srce.hr
-
 # list of VOs for which you wish to run checks
 VO = dteam
 
 #
 # Some optional config
 #
-#BDII = bdii.srce.hr
-#GLITE_VERSION=3.1.0
```
Comment (themiszamani) [inline config/ncg.conf:34]: what do they mean? please explaun at least  ENABLE_UNICORE_PROBES why we need it and what is myproxy
NAGIOS_SERVER = nagios.fqdn
ENABLE_UNICORE_PROBES=1
MYPROXY_SERVER = myproxy.fqdn

File: config/ncg.conf [hunk 2]
```diff
@@ -42,91 +32,77 @@ VO = dteam
...
-    NRPE_OUTPUT_DIR = /etc/nagios/nrpe/
-    RESOURCES_OUTPUT_DIR = /etc/nagios
-
-    # Modify following to values used for creating MyProxy
-    # credential with command myproxy-init
-    # myproxy-init argument -k
-    MYPROXY_NAME=NagiosRetrieve
-    # myproxy-init argument -l
-    MYPROXY_USER=nagios
+    NAGIOS_ROLE = ngi
```
Comment (themiszamani) [inline config/ncg.conf:35]: explain all different values
NAGIOS_ROLE = ngi if not ngi what else can we put
when empty hosts is 1 and why?
    INCLUDE_EMPTY_HOSTS = 0
what is enable notifications? enable nagios notifications for whom
    ENABLE_NOTIFICATIONS = 0
what is SEND_TO_DASHBOARD?
    SEND_TO_DASHBOARD = 0
what is CHECK_HOSTS?
    CHECK_HOSTS = 0

    TENANT = ARGO-TEST
robot certificates. Why do we need them 
    ROBOT_CERT=/etc/nagios/globus/robocert.pem
    ROBOT_KEY=/etc/nagios/globus/robokey.pem
  when do we set   USE_ROBOT_CERT to 1
  USE_ROBOT_CERT=1
what is SEND_TO_MSG? When do we use it
    SEND_TO_MSG = 0

File: config/ncg.conf [hunk 3]
```diff
@@ -42,91 +32,77 @@ VO = dteam
...
-    #       | xmllint --format -
-    # SAM_VOS=OPS
+    GOCDB_ROOT_URL=https://goc.egi.eu/gocdbpi
+    ENABLE_UNICORE_PROBES=$ENABLE_UNICORE_PROBES
+    BDII_HOST=bdii.fqdn
+    INCLUDE_PROXY_CHECKS = 1
+    INCLUDE_MSG_CHECKS_SEND = 0
   </Active>
-
+  <LDAP>
```
Comment (themiszamani) [inline config/ncg.conf:97]: when do we need ldap?

File: config/ncg.conf [hunk 4]
```diff
@@ -42,91 +32,77 @@ VO = dteam
...
-    #
-    # Option SAM_VOS filters VOs for which SAM results
-    # are gathered
-    # WARNING: VO names are case sensitive, fetch available
-    # VOs with following command (from Nagios server):
-    #  wget -q -O - http://lcg-sam.cern.ch:8080/same-pi/vo_names.jsp \
-    #       | xmllint --format -
-    # SAM_VOS=OPS
+    GOCDB_ROOT_URL=https://goc.egi.eu/gocdbpi
+    ENABLE_UNICORE_PROBES=$ENABLE_UNICORE_PROBES
```
Comment (themiszamani) [inline config/ncg.conf:92]: What are these values. 
    ENABLE_UNICORE_PROBES=$ENABLE_UNICORE_PROBES
    BDII_HOST=bdii.fqdn
    INCLUDE_PROXY_CHECKS = 1
    INCLUDE_MSG_CHECKS_SEND = 0

**Comment by eimamagi** [comment]
I will reopen this one later on.

**Pull request closed without merging**